### PR TITLE
Issue1: fixed

### PR DESCRIPTION
**Pull request by skozubenko1**
shows file changes made to correct issue 1 & 2.
Commit section addressing issue 3
Tests for randomize were added.

### CODE REVIEW

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -24,7 +24,8 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            //issue2-fixed!!!
+            return (numberCorrectAnswers * 1.0 / numberOfQuestions * 100) + "%";
```
Comment (mmwoodfo) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs]: I believe that this is supposed to be fixed in a different PR.

Comment (skozubenko1) [reply PrincessBrideTrivia/PrincessBrideTrivia/Program.cs]: I have it in a different pull request. Accidentally merged them while figuring out how to use git. Thank you!
